_layouts/base.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
- 		<link id="defaultIcon1" rel="shortcut icon" href="{{ site.baseurl }}assets/favicon_06.png" />
+        <link id="defaultIcon1" rel="shortcut icon" href="/assets/favicon.png" />
```

Comment:
Why is site.baseurl removed?

Why is there no file assets/favicon.png yet you link to it here?

Reply:
I will revert the changes in `_layouts/base.html` to the previous version. 
Since there is no `/favicon.png`, should I refer to `/assets/favicon_06.png` here instead as before?

Reply:
Don't mind either way, just that it should refer to something that exists.